.tekton/pipeline-service-test.yaml

```diff
@@ -31,6 +31,8 @@ spec:
       - name: shared-workspace
     tasks:
       - name: produce-cluster-name
+        runAfter:
+          - "fetch-repository"
```

Comment:
Is there a reason this has to be run after fetch-repository?

Reply:
It makes a bit more sense when looking at flow in the ui, as `delete-clusters` does not depend on that task.